testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1448,3 +1448,26 @@ def server_secret
   command = "sed -i '$d' /etc/hosts && sed -i '$d' /etc/hosts"
   $server.run(command)
 end
+
+# select an item from any dropdown
+And(/^I select "(.*?)" from "([^"]*)" dropdown/) do |selection, label|
```

Comment:
Wondering, can't https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/step_definitions/navigation_steps.rb#L181 be used instead of this step?

Reply:
I was wondering aswell and tried to use this definition. Without success sadly. *If* I remember correcty, it gave me a Capybara Element not Found.

Reply:
Maybe the element is not visible without scrolling, that could cause the step to fail, not sure
If that's the case, that would be a good fix.